SYCL/ESIMD/api/functional/operators/operator_assignment_copy_core.cpp

```diff
@@ -0,0 +1,58 @@
+//==------- operator_assignment_copy_core.cpp  - DPC++ ESIMD on-device test
+//          ----------------------------------------------------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu, level_zero
+// XREQUIRES: gpu
+// TODO gpu and level_zero in REQUIRES due to only this platforms supported yet.
+// The current "REQUIRES" should be replaced with "gpu" only as mentioned in
+// "XREQUIRES".
+// UNSUPPORTED: cuda, hip
+// RUN: %clangxx -fsycl %s -fsycl-device-code-split=per_kernel -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+//
+// Test for simd copy assignment operator.
+// The test creates source simd instance with reference data and invokes copy
+// assignment operator from source to destination simd instance. It is expected
+// for destination simd instance to store a bitwise same data as the reference
+// one.
+
+#include "operator_assignment.hpp"
+
+using namespace sycl::ext::intel::experimental::esimd;
+using namespace esimd_test::api::functional;
+
+// Descriptor class for the case of calling constructor in initializer context.
```

Comment:
Update please as discussed [earlier](https://github.com/intel/llvm-test-suite/pull/762#discussion_r791622390)

Reply:
I've applied this comment [here](https://github.com/intel/llvm-test-suite/pull/762/commits/92df51c1386dfd2e52709190f04f2c7baaec113d#diff-5ae6688531ef20b0d69bf7dd54176b9516394d4266983c2876a387324a402c15R29)